### ISSUE TITLE
(regression) localized about screen shows code and entities instead of umlauts

### ISSUE THREAD

**Issue by cadeyrn** [mozilla-mobile/focus-android#5308]
### Steps to reproduce

0. set your language to German
1. open the about screen

### Expected behavior

The "About" content looks good.

### Actual behavior

![Screenshot_20210901-225634](https://user-images.githubusercontent.com/1100614/131744446-034ce077-935f-4cec-976e-c0becf98d7e4.jpg)

It's probably a regression from #5242. /cc @mcarare 

### Device information

* Android device: OnePlus 7T Pro McLaren, Oxygen OS 11 (Android 11)
* Focus version: main branch revision 76e7ff91144720de747bf2ffde9440ba63c2fd8e

**Comment by mcarare** [comment]
@Delphine Any reason this string has been updated in such a manner here: https://pontoon.mozilla.org/de/focus-for-android/mozilla-mobile/focus-android/app/src/main/res/values/strings.xml/?search=about_content&string=224023 ?

**Comment by Delphine** [comment]
@mcarare Ok I've updated the string in Pontoon. This looks like a bug in the toolchain, and seems to only affect German, from what I can see at this point (so not 100% sure either).
[There are other instances](https://transvision.flod.org/?recherche=xml&repo=android_l10n&sourcelocale=en-US&locale=de&search_type=strings) where this happens in German, and I'm going to clean it up manually now.
CC @mathjazz in case he can help identify what's going on here 🤞

**Comment by mcarare** [comment]
Thanks, @Delphine!

**Comment by mathjazz** [comment]
Hey, it looks like the issue was caused by copying a bad suggestion from the Translation Memory.

The entry was imported into the Translation Memory as part of bootstrapping the [Firefox for TV](https://github.com/mozilla-l10n/firefoxtv-l10n/pull/1) project back in 2018. It seems like we noticed the bad translations and deleted them, but forgot to delete the corresponding TM entries.

I did that now to avoid bad copying and pasting in the future.
